### PR TITLE
Inventory service

### PR DESCRIPTION
**Pull request by abdallamaged95**
Inventory Service done without testing or validation

### CODE REVIEW

File: services/inventory/src/main/java/com/bazaar/inventory/InventoryApplication.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+		org.springframework.boot.autoconfigure.security.servlet.SecurityAutoConfiguration.class}
+)
+public class InventoryApplication {
+
+	public static void main(String[] args) {
+		SpringApplication.run(InventoryApplication.class, args);
+	}
+
+	@Bean
+	public CommandLineRunner myRunner(ProductServiceImpl productService) {
```
Comment (oeh808) [inline services/inventory/src/main/java/com/bazaar/inventory/InventoryApplication.java:19]: Remove

Comment (abdallamaged95) [reply services/inventory/src/main/java/com/bazaar/inventory/InventoryApplication.java:19]: oky doky

File: services/inventory/src/main/java/com/bazaar/inventory/controller/CategoriesController.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                        categoryMapper.toCategory(categoryDto)
+                )
+        );
+    }
+
+    @DeleteMapping("/{categoryId}")
+    @ResponseBody
+    @ResponseStatus(HttpStatus.OK)
+    public String deleteProduct(@PathVariable Long categoryId) {
+        categoryService.delete(categoryId);
```
Comment (oeh808) [inline services/inventory/src/main/java/com/bazaar/inventory/controller/CategoriesController.java:65]: Return String from Service

File: services/inventory/src/main/java/com/bazaar/inventory/controller/CategoriesController.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Controller
+@RequestMapping("/api/v1/categories")
+@AllArgsConstructor
+public class CategoriesController {
```
Comment (oeh808) [inline services/inventory/src/main/java/com/bazaar/inventory/controller/CategoriesController.java:17]: Update names from Product to Category

File: services/inventory/src/main/java/com/bazaar/inventory/service/CategoryServiceImpl.java [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+import jakarta.transaction.Transactional;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+
+import java.util.List;
+import java.util.Optional;
+@Service
+@AllArgsConstructor
+public class CategoryServiceImpl implements CategoryService{
```
Comment (oeh808) [inline services/inventory/src/main/java/com/bazaar/inventory/service/CategoryServiceImpl.java:17]: Remove unnecessary @Transactionals

**Review by oeh808 — CHANGES_REQUESTED** [review]
Check Comments

**Review by abdallamaged95 — COMMENTED** [review]
Completed Change Requests

**Pull request merged**